client/src/unifycr.c

```diff
@@ -1679,21 +1686,30 @@ static int unifycr_abtoull(char *str, unsigned long long *val)
 
 static int unifycr_init(int rank)
 {
-    if (! unifycr_initialized) {
+    if (!unifycr_initialized) {
+
+		/* unifycr DEBUG level default is zero */
+        unifycr_debug_level = 0;
```

Comment:
Indentation looks off here. Probably from mixing tabs and spaces.

I use this in my .vimrc to make that easier to spot:

```
set list            " * Turn on printing of invisible chars
set listchars=tab:>-,trail:! " * Customize the characters used to print the
                    "   invisible chars
```

Reply:
@nedbass yes, thanks. I can see the tabs now!